src/convert/declarations.test.ts

```diff
@@ -225,21 +228,22 @@ describe("transform declarations", () => {
     class Impl extends Base<Function, {}> {}
     `;
     const expected = dedent`
-    import type {$TSFixMeFunction} from 'v2/core/util/flowCompat';
-    class Base<P = any, S = any> {}
-    class Impl extends Base<$TSFixMeFunction, Record<any, any>> {}
+    import type {$TSFixMeFunction, $TSFixMeAny} from 'v2/core/util/flowCompat';
```

Comment:
Nice to have these tests and confirm that it handles multiple type imports correctly